Library/Formula/ghi.rb

```diff
@@ -7,6 +7,8 @@ class Ghi < Formula
 
   bottle :unneeded
 
+  depends_on "pygments.rb" => :ruby
```

Comment:
`:ruby` doesn't always mean Homebrew's Ruby. It's whichever Ruby happens to be first in the `$PATH` and suitable. It uses my Ruby via `rbenv`, for example. Can you detail what you're seeing more?